applications/stepswitch/src/stepswitch_local_extension.erl

```diff
@@ -30,6 +30,8 @@
                ,response_queue :: api_binary()
                ,queue :: api_binary()
                ,timeout :: reference()
+               ,call_id :: api_binary()
+               ,call_ids :: api_binaries()
```

Comment:
@lazedo Why track the call_ids, is this for future use?  I looked for where you use this and didn't initially see anything other than collection.